utils/guidelines_xslt/odd2html/htmlFile.xsl

```diff
@@ -248,7 +248,7 @@
                             </div>
                         </div>
                         
-                        <script type="text/javascript">
+                        <script>
```

Comment:
This is a separate issue – please remove and commit independent from the other change